### PR TITLE
feat: add workspace cli

### LINKED ISSUE #604: Feat: add workspace config management commands
### What would you like to be added?
Add workspace config management commands as follows:
- [ ] Basic workspace operations (init, list, get)
- [ ] Add one command to support validating workspace configs

### Why is this needed?

Workspace Configs are composed of many blocks and override logic. We need a command to help users to validate and view their env configs.

### PR DESCRIPTION
**Pull request by healthjyk**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

add workspace cli

#### Which issue(s) this PR fixes:

Fixes #604

### CODE REVIEW

File: pkg/cmd/workspace/create/cmd.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+func NewCmd() *cobra.Command {
+	var (
+		short = i18n.T(`Create a new workspace`)
+
+		long = i18n.T(`
+		This command creates a workspace with specified name and configuration file, where the file must be in the YAML format.`)
+
+		example = i18n.T(`
+		# Create a new workspace
+		kusion workspace create dev -f dev.yaml`)
```
Comment (liu-hm19) [inline pkg/cmd/workspace/create/cmd.go:20]: It seems that the `kusion workspace create` method does not provide the functionality of workspace scaffolding templates? Is there any plan to support it in the future?

File: pkg/cmd/cmd.go [hunk 2]
```diff
@@ -90,6 +92,7 @@ func NewKusionctlCmd(o KusionctlOptions) *cobra.Command {
 		{
 			Message: "Configuration Commands:",
 			Commands: []*cobra.Command{
+				workspace.NewCmd(),
```
[the remark this answers is not in the document]
Comment (healthjyk) [reply pkg/cmd/cmd.go:95]: Cause the pkg name is the same as the sub command name,  thus using `NewCmd` as func name is better than `NewWorkspaceCmd`. Maybe other new funcs need to change their names.

File: pkg/cmd/workspace/update/cmd.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	"kusionstack.io/kusion/pkg/cmd/util"
+	"kusionstack.io/kusion/pkg/util/i18n"
+)
+
+func NewCmd() *cobra.Command {
+	var (
+		short = i18n.T(`Update a workspace configuration`)
+
+		long = i18n.T(`
+		This command updates a workspace configuration with specified configuration file, where the file must the YAML format.`)
```
[the remark this answers is not in the document]
Comment (healthjyk) [reply pkg/cmd/workspace/update/cmd.go:16]: That's better, I'll fix it.

**Review by liu-hm19 — APPROVED** [review]
lgtm

**Pull request merged**